erdjs/src/smartcontracts/codec/tuple.ts

```diff
@@ -0,0 +1,27 @@
+import { Struct, TupleType, Tuple } from "../typesystem";
+import { BinaryCodec } from "./binary";
+import { StructBinaryCodec } from "./struct";
+
+export class TupleBinaryCodec {
+    private structCodec: StructBinaryCodec;
+
+    constructor(binaryCodec: BinaryCodec) {
+        this.structCodec = new StructBinaryCodec(binaryCodec);
```

Comment:
Optional: is it possible to use the `structCodec` already initialized in `binaryCodec`? Don't know if it's a good idea overall. The `structCodec` is indeed private inside `binaryCodec`.

Maybe it's possible to rewrite `TupleBinaryCodec` to inherit `StructCodec`?